pkg/rpcclient/rpc.go

```diff
@@ -974,9 +974,17 @@ func (c *Client) GetRawNotaryPool() (*result.RawNotaryPool, error) {
 }
 
 // GetBlockNotifications returns notifications from a block organized by trigger type.
-func (c *Client) GetBlockNotifications(blockHash util.Uint256, filters ...*neorpc.NotificationFilter) (*result.BlockNotifications, error) {
+func (c *Client) GetBlockNotifications(blockHash util.Uint256, filter ...*neorpc.NotificationFilter) (*result.BlockNotifications, error) {
```

Comment:
Why do we ever have filters specified via `...`? Change it to `filter *neorpc.NotificationFilter` to match WSClient's notation of `Receive*` functions. It's an incompatible change, but it's trivial to handle on the user's side.

@roman-khimov ACK?

Reply:
ACK